browser/payments/BUILD.gn

```diff
@@ -1,23 +1,13 @@
-declare_args() {
-  # anonize doesn't currently build on windows
-  brave_payments_enabled = is_mac || is_linux
-}
-
-config("external_config") {
-  visibility = [ ":payments" ]
-
-  defines = []
-  if (brave_payments_enabled) {
-    defines += [ "BRAVE_PAYMENTS_ENABLED" ]
-  }
-}
+import("//brave/browser/payments/buildflags/buildflags.gni")
 
 source_set("payments") {
-  public_configs = [
-    ":external_config"
+  public_deps = [
+    "buildflags",
   ]
 
   sources = [
+    "content_site.cc",
+    "content_site.h",
```

Comment:
Build failed with below log.
```
../..\brave/browser/payments/content_site.h(32,14):  error: no template named 'vector' in namespace 'std'
typedef std::vector<const ContentSite> ContentSiteList;
        ~~~~~^
../../brave/browser/payments/content_site.cc(6,10):  fatal error: 'bat/ledger/publisher_info.h' file not found
#include "bat/ledger/publisher_info.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Not sure but above two files should be moved into `brave_payments_enabled`?

Reply:
yep, I'll fix

Reply:
https://github.com/brave/brave-core/pull/294